site/specs-temp/numbers.json

```
@@ -30400,6 +30496,133 @@
             }
           }
         }
+      },
+      "ShortCode": {
+        "type": "object",
+        "properties": {
+          "ShortCodeNumber": {
+            "type": "integer"
+          },
+          "AccountId": {
+            "type": "integer"
+          },
+          "SipPeerId": {
+            "type": "integer"
+          },
+          "Country": {
+            "type": "string"
+          },
+          "DateFirstOrdered": {
+            "type": "string"
+          },
+          "LeaseStartDate": {
+            "type": "string"
+          },
+          "LeaseEndDate": {
+            "type": "string"
+          },
+          "Status": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "INACTIVE",
+              "SUSPENDED",
+              "EXPIRED"
+            ],
+            "default": "INACTIVE"
+          },
+          "AttStatus": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "INACTIVE"
+            ],
+            "default": "INACTIVE"
+          },
+          "TmoStatus": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "INACTIVE"
+            ],
+            "default": "INACTIVE"
+          },
+          "VerizonStatus": {
+            "type": "string",
+            "enum": [
+              "ACTIVE",
+              "INACTIVE"
+            ],
+            "default": "INACTIVE"
+          },
+          "Type": {
+            "type": "string",
+            "enum": [
+              "RANDOM",
+              "VANITY"
+            ]
+          },
+          "CustomerManagedLease": {
+            "type": "boolean"
+          }
+        }
+      },
+      "ShortCodeResponseList": {
```

```suggestion
      "shortCodeResponseList": {
```
like Brian mentioned, there are a lot of uncaught camelCase issues in this spec, but going forward we should try to make these schema names conform to the standard.